processor/src/main/java/org/mapstruct/ap/internal/model/source/SubclassMappingOptions.java

```diff
@@ -84,7 +86,7 @@ private static boolean isConsistent(SubclassMappingGem gem, ExecutableElement me
                         targetSubclass.toString() );
             isConsistent = false;
         }
-        subclassValidator.isInCorrectOrder( method, gem.mirror(), targetSubclass );
+        isConsistent = isConsistent & subclassValidator.isValidUsage( method, gem.mirror(), targetSubclass );
```

Comment:
I personally always have problems with the bitwise operator. Can we use something else here perhaps?

e.g.
```
return subclassValidator.isValidUsage( method, gem.mirror(), targetSubclass ) && isConsistent;
```

or even a simple if

```
if (!subclassValidator.isValidUsage( method, gem.mirror(), targetSubclass )) {
    isConsistent = false;
}
```